lib/core/ui/context_menu_item.dart

```diff
@@ -2,14 +2,11 @@ import 'package:flutter/material.dart';
 
 class ContextMenuItem extends StatelessWidget {
   const ContextMenuItem({
-    @required this.title,
-    @required this.icon,
-    @required this.onTap,
-    Key key,
-  })  : assert(title != null),
-        assert(icon != null),
-        assert(onTap != null),
-        super(key: key);
+    required this.title,
+    required this.icon,
```

Comment:
```suggestion
    this.icon = Icons.warning,
```

Reply:
Why have you suggested this icon here? @mukireus

Reply:
I agree with Muhammet because there was a control below. If the icon is null, puts Icons.warning.